code/game/objects/structures/crates_lockers/closets/secure/medical_lockers.dm

```diff
@@ -204,16 +204,19 @@
 	req_access = list(ACCESS_PARAMEDIC)
 
 /obj/structure/closet/secure_closet/paramedic/populate_contents()
+	new /obj/item/radio/headset/headset_med/para(src)
 	new /obj/item/mod/control/pre_equipped/rescue(src)
```

Comment:
Perhaps add a Holster Module for the Rescue MOD? Would make carrying the Plasma Cutter for the Paramedic more convenient and the Syringe Gun for the CMO - double positive QoL IMO.

Reply:
Though you could also just have it be part of the upgrade path if you so choose - either or would be fine.

Reply:
After some thinking, while a holster module would be very helpful for both roles I'd be concerned about making Paramedic players think they're encouraged to carry around weapons- given some incidents we have with Paramedic players validhunting.

It WOULD be helpful for them, but I think they can go to robotics if they want a holster module.